src/App.tsx

```diff
@@ -1,61 +1,38 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types';
+import { TodoForm } from './components/TodoForm/TodoForm';
 
 export const App = () => {
+  const enrichTodo = (todo: Todo) => ({
+    ...todo,
+    user: usersFromServer.find(user => user.id === todo.userId),
+  });
+
+  const todoData = todosFromServer.map(enrichTodo);
+
+  const [todos, setTodos] = useState<Todo[]>(todoData);
+
+  const addTodo = (todo: Todo) => {
+    const newTodo: Todo = {
+      ...todo,
+      id: Math.max(...todos.map(t => t.id!)) + 1,
```

Comment:
Ensure that all todos have an `id` defined. The use of `t.id!` assumes that `id` is always present, which might lead to runtime errors if any todo lacks an `id`. Consider adding a check or default value.